src/modules/infobox/components/styling/DrawingStyleMediaLink.vue

```diff
@@ -0,0 +1,138 @@
+<script setup>
+import { computed, defineEmits, ref, toRefs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+import { isValidUrl } from '@/utils/utils'
+
+const i18n = useI18n()
+const generatedMediaLink = ref('')
+const linkDescription = ref('')
+const props = defineProps({
+    mediaType: {
+        type: String,
+        required: true,
+    },
+    urlDescription: {
+        type: String,
+        required: true,
+    },
+    extraUrlDescription: {
+        type: String,
+        default: null,
+    },
+})
+
+const urlValid = computed(() => {
+    return isValidUrl(generatedMediaLink.value)
+})
+
+const urlDescriptionValid = computed(() => {
+    return !extraUrlDescription.value || linkDescription.value
+})
```

Comment:
I think this would be better below the defineEmits dans transformation of props into refs